doc/devdocs/disk-usage-footprint.md

```diff
@@ -1,40 +1,71 @@
 
-This adds .NET to the list of dependencies we are shipping self-contained (together with WinAppSDK and VC Redistributable) in PowerToys by using the hard-links technology to share the dlls between the modules in a way that saves disk space when compared to having actual copies for each module. Our main motivation for doing this is to eventually reach a point where PowerToys can be installed without requiring UAC elevation and without installing additional dependencies like .NET.
+## Why do this work?
 
-The main goal of this document is to show how PowerToys has a lower disk space impact than shown in File Explorer.
+Our main motivations for doing are the following:
 
-File Explorer treats hard-links the same as "original"/regular files/directories (https://learn.microsoft.com/en-us/troubleshoot/windows-server/backup-and-storage/disk-space-problems-on-ntfs-volumes#other-ntfs-features-that-may-cause-file-allocation-confusion), not as links. This results in File Explorer reporting size of PowerToys installation directory bigger than it is (more than 2GB). Shown bellow is real disk usage footprint of PowerToys v0.66 obtained by installing PowerToys on empty disk:
+1. Be able to install PowerToys without UAC elevation
+2. PowerToys has a reduced disk space when components are shared.
+3. Reduce additional downloads for dependencies
 
-# Steps
+## Current self-contained runtimes 
 
-## Empty disk details
+These dll's installed in `<PowerToysInstallDir>/dll/<Item>` folder structure.
+
+- [Windows Application SDK runtime](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads)
+- [.NET Desktop Runtime 7](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
+- [Microsoft Visual C++ Runtime](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170)
+
+## So why does PowerToys have a larger footprint now?
+
+Before the hard-link work was done, each of these runtimes had to be installed so there was more items being installed outside just the PowerToys install directory. This space would not have been accounted, 
+
+File Explorer treats hard-links the same as "original"/regular files/directories (https://learn.microsoft.com/en-us/troubleshoot/windows-server/backup-and-storage/disk-space-problems-on-ntfs-volumes#other-ntfs-features-that-may-cause-file-allocation-confusion), not as links. This results in File Explorer reporting size of PowerToys installation directory bigger than it is (more than 2GB). 
+
+## How much space is actually being used
+
+As of v0.66 of PowerToys, the installed footprint is ~660MB. 
+
+When we were prototyping and validating, here was our running table of results and why we took the route we did.
+
+| Moment | Installer size | App size | Installation dir size | Real size | Deps downloaded and installed during PT install | Total (installer size + real size + deps downloaded and installed during PT install) |
```

Comment:
Would be nice to write the word "dependencies" at least once, full out, here or there...